### PR TITLE
bpo-23894: Make rb'' strings work in lib2to3

### PR DESCRIPTION
**Pull request by ambv**
This partially solves bpo-23894. A separate PR coming later to support f-strings.

### CODE REVIEW

File: Lib/lib2to3/pgen2/tokenize.py [hunk 1]
```diff
@@ -74,10 +74,11 @@ def maybe(*choices): return group(*choices) + '?'
 Single3 = r"[^'\\]*(?:(?:\\.|'(?!''))[^'\\]*)*'''"
 # Tail end of """ string.
 Double3 = r'[^"\\]*(?:(?:\\.|"(?!""))[^"\\]*)*"""'
-Triple = group("[ubUB]?[rR]?'''", '[ubUB]?[rR]?"""')
+_litprefix = r"(?:[uUrRbB]|[rR][bB]|[bBuU][rR])?"
+Triple = group(_litprefix + "'''", _litprefix + '"""')
 # Single-line ' or " string.
-String = group(r"[uU]?[rR]?'[^\n'\\]*(?:\\.[^\n'\\]*)*'",
-               r'[uU]?[rR]?"[^\n"\\]*(?:\\.[^\n"\\]*)*"')
```
Comment (ambv) [inline Lib/lib2to3/pgen2/tokenize.py:80]: "String" didn't previously include `b` but this is suspicious to me. Why wouldn't it? That way `b''` literals were only PseudoTokens (via ContStr), not actual Tokens (via String).

**Review by ned-deily — APPROVED** [review]
Thanks for adding the test: LGTM.  As far as the tokenizer changes, I'm not familiar enough with lib2to3 to provide a useful review.  Assuming other reviewers have no objections, I'm fine for this going into 3.6.2rc.

**Comment by ambv** [comment]
Tests are failing because I used an f-string in the test and lib2to3 is parsing that test file as part of fixers. f-strings are not supported yet in lib2to3. Switching to using a classic `str.format` fixes it. I'm updating the PR now.

UPDATE: as expected, tests pass now.

File: Lib/lib2to3/pgen2/tokenize.py (hunk 1 above)
Comment (gvanrossum) [reply Lib/lib2to3/pgen2/tokenize.py:80]: LG

**Comment by vstinner** [comment]
I was requested for a review, but after I completed my review, I see that the change was already merged :-D It's fine, don't worry. I just want to give my post-commit LGTM! Nice change. Thanks @ambv for taking care of 2to3 ;-)

**Pull request merged**